### ISSUE TITLE
Add tag for `lts` releases

### ISSUE THREAD

**Issue by bluesmoon** [docker-library/julia#92]
Hi folks,

Would it be possible to add a shared tag for `lts` releases of Julia? This would always map to the latest LTS release, which should be 1.10.5 (or 1.10) at the time of this writing.

In our workflow we only use LTS releases, so it would help to have our CI pull from `julia:lts` rather than having to figure out if `julia:latest` is LTS or not.

Thank you.

**Comment by tianon** [comment]
Oh, we do _not_ hand-maintain "latest", that's automatic based on which versions we currently have.  Hmm.

**Comment by tianon** [comment]
I wonder if we could convince upstream folks to add an `lts` boolean to https://julialang.org/downloads/#json_release_feed :eyes: :heart:

**Comment by tianon** [comment]
No, "latest" will always point to the highest numeric GA release (so until there's a 1.12 or 2.0 that's GA, it will point to whatever the latest patch release of 1.11 is).

**Comment by fredrikekre** [comment]
> I wonder if we could convince upstream folks to add an lts boolean to https://julialang.org/downloads/#json_release_feed 👀 ❤️

See https://github.com/JuliaLang/VersionsJSONUtil.jl/issues/2. I don't think anyone would object, just nobody has implemented it.
